reV/config/pipeline.py

```diff
@@ -86,18 +86,28 @@ def _check_dirout_status(self):
                     raise PipelineError(msg)
 
     @property
-    def pipeline_steps(self):
+    def pipeline(self):
         """Get the pipeline steps.
 
         Returns
         -------
-        _pipeline_steps : list
+        pipeline : list
             reV pipeline run steps. Should be a list of (command, config)
             pairs.
         """
 
         return self['pipeline']
 
+    @property
+    def logging(self):
```

Comment:
Should we move this to the base config class? Seems useful!

Reply:
Pipeline and batch use these logging dictionaries but everything else uses a combination of log_level at the base level and a logging directory in the directories namespace. I'm not totally opposed but it would require a full rework of all the config organizations and would break backwards compatibility.

Reply:
gotcha, I think this is fine then even if its in two places.